docs/manual-curation/step2-manual-curation.md

```diff
@@ -3,14 +3,15 @@
 The goals of the manual curation:
 * All traits which are linked to NT expansion (nucleotide repeat expansion) variants must be curated. Those are marked as "NT expansion" in the “Notes” column.
 * All traits with occurrence ≥ **10** must be curated. Additionally, if there are less than **200** such terms, then the top 200 terms must be curated.
+* _Suggested previous mapping_ traits should be checked for any terms that have become obsolete since the last iteration. This can be done by filtering then searching for the string EFO\_OBSOLETE
```

Comment:
I think we should have a better way for the curators and reviewers to check this bit. Something like a colour code or tag somewhere that we could easily identify.

Reply:
After all it's just parsing the end of the previous mapping

Reply:
Agreed. @tcezard & I had a short discussion in which we think that the process I follow for manual curation should be documented step by step. We have scheduled to work on this task and will add this comment to the created JIRA ticket